src/libdrakvuf/libdrakvuf.h

```diff
@@ -424,6 +429,8 @@ addr_t drakvuf_exportksym_to_va(drakvuf_t drakvuf,
 addr_t drakvuf_exportsym_to_va(drakvuf_t drakvuf, addr_t process_addr,
                                const char* module, const char* sym);
 
+addr_t drakvuf_export_linux_sym_to_va(drakvuf_t drakvuf, drakvuf_trap_info_t* info, vmi_pid_t pid, const char* lib);
```

Comment:
Ah wait, you are actually just returning the base address for `libc` here, not actually looking up a specific function's address within libc? In that case, this function should be `drakvuf_get_module_base_addr`.